javascript-arrays/main.js

```diff
@@ -0,0 +1,27 @@
+const colors = ['red', 'white', 'blue'];
+
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+
+const americaString =
+  'America is ' + colors[0] + ', ' + colors[1] + ', and ' + colors[2] + '.';
+console.log(americaString);
+
+colors[2] = 'green';
+
+const mexicoString =
+  'Mexico is ' + colors[0] + ', ' + colors[1] + ', and ' + colors[2] + '.';
+console.log(mexicoString);
+
+console.log(colors);
+
+const students = ['Mary', 'Tom', 'Andy', 'Sam'];
+
+const numberOfStudents = students.length;
+console.log('There are ' + numberOfStudents + ' students in the class.');
+
+const lastIndex = [numberOfStudents - 1];
```

Comment:
This should just be `const lastIndex = numberOfStudents - 1;`. By including the `[]` around it, you're technically creating another array. This should just be a number.